iXBRLViewerPlugin/viewer/src/less/summary.less

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#inspector .summary {
+  .body {
+    sup {
+      font-size: 0.9rem;
+    }
+
+    .total-facts-title,
+    .files-summary-title {
+      font-weight: bold;
+    }
+
+    .file-summary-container {
+      display: flex;
+      flex-flow: row wrap;
+
+      .files {
+        flex: 1 1 auto;
+      }
+    }
+
+    table {
+      width: 100%;
+      border-collapse: collapse;
+
+      td,
+      th {
+        padding: 0.2rem;
+        text-align: left;
+      }
+
+      thead {
+        th {
+          word-break: break-word;
```

Comment:
So this isn't a critical UI issue, but here are some options on getting the summary table to work a bit narrower:
Apparently `word-break: break-word;` [is deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), so the headers are breaking mid-word. The alternative would be:
```suggestion
          white-space: break-spaces;
```
But that means only "Primary Items" breaks a line, so the minimum table width is still larger than the starting inspector width. We could do one or both of these options:

1. Wrap the table in `<div style="overflow-x:auto">...</div>` so that when the table does exceed the inspector width, it can still be fully viewed.
2. Abbreviate some/all of the headers (NS, PI, Dim, Mem, etc.)

![Screenshot 2023-05-11 at 1 09 37 PM](https://github.com/Workiva/ixbrl-viewer/assets/105066394/87618fe6-38c8-4738-8576-bf21e683e412)
![Screenshot 2023-05-11 at 1 09 57 PM](https://github.com/Workiva/ixbrl-viewer/assets/105066394/d61e0246-6907-45df-824d-c27deef5003e)
![Screenshot 2023-05-11 at 1 11 03 PM](https://github.com/Workiva/ixbrl-viewer/assets/105066394/deddf38a-7035-4140-9d29-fdf3c05f492c)
![Screenshot 2023-05-11 at 1 13 42 PM](https://github.com/Workiva/ixbrl-viewer/assets/105066394/7ce93a5d-65b4-463b-be4c-0e25edbfb004)